src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,224 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected String by;
+    protected LocalDate date;
+    protected LocalTime time;
+
+    /**
+     * Gets the description of the deadline
+     * @param s user input
+     * @return the description of the deadline
+     */
+    public static String getDescription(String s) {
```

Comment:
could possibly write 'description' rather than 's'?

Reply:
Yes. It should "userInput".